pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -485,6 +485,9 @@ pyramid_oereb:
     sort_within_themes_method: pyramid_oereb.standard.hook_methods.plr_sort_within_themes
     # Example of a specific sorting method:
     # sort_within_themes_method: pyramid_oereb.contrib.plr_sort_within_themes_by_type_code
+    # Redirect configuration for type URL. You can use any attribute of the real estate RealEstateRecord
```

Comment:
I would think only identifiers like EGRID or IDENTDN&NUMBER should be possible or am I wrong?

Reply:
I am not sure. The whole real estate object is passed to the method, so all attributes are available to parameterize the URL. Maybe some canton needs also the type of the real estate. But I think in most cases, the `egrid` will be enough.

What do the others think? Should we restrict the available attributes to `egrid`, `identdn` and `number`?

Reply:
I think EGRID is enough because we have a service to obtain it with other identifiers.